controllers/subscriber/subscriberControllers.js

```diff
@@ -81,6 +81,13 @@ export class SubscriberRegistryController extends BaseController {
 		return this._subscribers;
 	}
 
+	handleSubscriberChange() {
+		if (!this._subscribers || this._subscribers.size === 0) return;
+		if (!this._options.handleSubscriberChange) return;
+
+		this._options.handleSubscriberChange(this._subscribers);
+	}
+
```

Comment:
It's been a while so maybe what I'm imagining doesn't work how I'd expect.  But what I was thinking was what you have now but with this deleted, and `handleSubscriberChange: this.handleSubscriberChange.bind(this)` deleted in `skeleton-group-mixin`.